statsmodels/regression/linear_model.py

```diff
@@ -52,11 +52,12 @@
 from statsmodels.tools.sm_exceptions import InvalidTestWarning
 
 # need import in module instead of lazily to copy `__doc__`
+from statsmodels.regression._prediction import PredictionResults
 from . import _prediction as pred
```

Comment:
Any reason not to include this in the following line?

Reply:
the second import is for the module